packages/tool-cache/RELEASES.md

```diff
@@ -1,5 +1,10 @@
 # @actions/tool-cache Releases
 
+### 2.0.0
+- Update to v2.0.0 of `@actions/http-client`
+- The type of the `headers` parameter in the exported function `downloadTool` has been narrowed from `{ [header: string]: any }` to `{ [header: string]: number | string | string[] | undefined; }` (that is, `http.OutgoingHttpHeaders`).
+    This is strictly a compile-time change for TypeScript consumers. Previous attempts to use a header value of a type other than those now accepted would have resulted in an error at run time.
```

Comment:
Note the major version bump.  This could be considered a bug fix, but I went with bumping the major version to be conservative.